ui/app/src/index.tsx

```diff
@@ -20,7 +20,7 @@ import App from './App';
 import { createTheme } from './styles/theme';
 import { SnackbarProvider } from './context/SnackbarProvider';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({ defaultOptions: { queries: { refetchOnWindowFocus: false } } });
```

Comment:
This just turns off the default `react-query` behavior that was causing PromQL queries, etc. to be refetched anytime you focus the browser window.

Reply:
cool ! That was simple at then. Nice catch !

Do you know what this behavior is the one by default ? It's a bit odd to have it by default, but probably I miss some context

Reply:
Yeah, I felt like it was a weird choice to have by default too, so don't think you're missing anything. The docs have a bit of an explanation of some of the defaults:

https://react-query.tanstack.com/guides/important-defaults